examples/fishjam-chat/components/VideosGrid.tsx

```diff
@@ -2,44 +2,64 @@ import {
   Track,
   VideoRendererView,
   Peer,
+  TrackType,
 } from '@fishjam-cloud/react-native-client';
-import React from 'react';
+import React, { useCallback } from 'react';
 import { FlatList, StyleSheet, View } from 'react-native';
 
 import { roomScreenLabels } from '../types/ComponentLabels';
 import { BrandColors } from '../utils/Colors';
 import Typo from './Typo';
 import { PeerMetadata } from '../types/metadata';
+import VADIcon from './VADIcon';
+import { AudioTrack } from '@fishjam-cloud/react-native-client/build/hooks/usePeers';
```

Comment:
**issue:** This import looks weird. I think it should be a part of `@fishjam-cloud/react-native-client`